snippets/ztp-patch-argocd-hub-cluster.adoc

```diff
@@ -22,9 +22,9 @@ Customize the patch file that you previously extracted into the `out/argocd/depl
 
 `registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel8:v2.9`
 
-|4.14, 4.15, 4.16
-|2.10
-|2.10
+|4.14, 4.15, 4.16, 4.17
+|2.10, 2.11
+|2.10, 2.11
 |RHEL 9
```

Comment:
The MCE version follows the ACM version which would include 2.12 for 4.17 (following this: https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.12/html-single/gitops/index#access-policy-gen-gitops)